lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -626,8 +699,10 @@ OrphanedNodesManager::variantSetDescriptors(const Ufe::Path& p)
         auto ancestor = Ufe::Hierarchy::createItem(path);
         auto usdAncestor = std::static_pointer_cast<UsdUfe::UsdSceneItem>(ancestor);
         auto variantSets = usdAncestor->prim().GetVariantSets();
+        auto setNames = variantSets.GetNames();
+        std::sort(setNames.begin(), setNames.end());
```

Comment:
Sort the variant set names to get a guaranteed consistent order for comparison purpose.